app/index.js

```diff
@@ -164,7 +164,23 @@ module.exports = yeoman.generators.Base.extend({
                 return;
             }
             this.installDependencies({
-                skipMessage: true
+                skipMessage: true,
+                callback: function (err) {
+                    var errString;
+                    if (err) {
+                        if (err.code === 'ENOENT') {
+                            errString = err.code + ': ' + err.path;
+                            if (err.path === 'bower') {
+                                errString += ' - cannot locate ' + err.path + '.' +
```

Comment:
So super minor but since we know `err.path === 'bower'` the errString can be defined more easily without the concatenation